### PR TITLE
Pass fewer arguments

### PR DESCRIPTION
**Pull request by croaky**
I spent a few minutes trying to figure out why the news feed isn't being
delivered to Campfire. I didn't figure out why and don't have time to
investigate further right now, but in the meantime, I saw a chance to refactor
a couple of things. This pull request represents that refactoring.

### CODE REVIEW

File: scripts/news.coffee [hunk 1]
```diff
@@ -8,28 +8,29 @@ class BotTimes
...
     @news.push(msg)
 
-  deliver: (robot) =>
-    robot.messageRoom('Everyone', @news...)
-    @news = @_newestEightyPercentOf(@news)
+  deliver: =>
+    @robot.messageRoom('Everyone', @news...)
+    @_trimToNewestEightyPercent()
 
     setTimeout () =>
```
Comment (jferris) [inline scripts/news.coffee:22]: You can just use `setTimeout @deliver, @FOUR_HOURS` now that you don't need arguments.

Comment (croaky) [inline scripts/news.coffee:22]: That changes the behavior somehow. When the robot brain boots, it "delivers" many times (perhaps 14400 \* 1000 times?).

Comment (jferris) [inline scripts/news.coffee:22]: @sgrif can I get a sanity check? Aren't these equivalent?

``` coffeescript
setTimeout @f, 1000

setTimeout =>
    @f()
  , 1000
```

(Given `f` is defined using `=>` and not `->`)

Comment (jferris) [inline scripts/news.coffee:22]: @croaky If it's not working, I wouldn't let this hold up the merge. I'm still interested in the difference, though.

**Pull request merged**